azure-sql/migration-guides/modernization.md

```diff
@@ -150,7 +150,7 @@ SQL Server CAL licenses with appropriate [license mobility](https://www.microsof
 
 #### What is Azure Hybrid Benefit (AHB)?
 
-Unique to Azure, Azure Hybrid Benefit (AHB) is a licensing benefit that allows you bringing your existing Windows Server and SQL Server licenses with Software Assurance (SA) to Azure. AHB can bring you up to 85% savings compared to pay-as-you-go pricing in Azure SQL, when combined with reservations savings and extended security updates.
+Unique to Azure, Azure Hybrid Benefit (AHB) is a licensing benefit that allows you bringing your existing Windows Server and/or SQL Server licenses (both Standard and Enterprise) with Software Assurance (SA) to Azure IaaS & Paas services. If you are planning to bring only existing SQL Server licenses, Azure Hybrid Benefits allow you bringing licences to Azure Data Factory SQL Server Integration Services, Azure SQL Database (both Elastic Pool and Single Database), SQL Managed Instances (General Purpose, Business Critical and Hyperscale) and Azure Arc-enabled SQL Managed Instance (both General Purpose and Business Critical). AHB can bring you up to 85% savings compared to pay-as-you-go pricing in Azure SQL, when combined with reservations savings and extended security updates. Azure Hybrid Benefit is not available in the serverless compute tier of Azure SQL Database. For more detailed information, how to use AHB with your own licenses see [Microsoft Azure Services Product Terms](https://www.microsoft.com/licensing/terms/productoffering/MicrosoftAzureServices/EAEAS)
```

Comment:
We generally do not discuss licensing details in Docs. This change will likely not be approved.

```suggestion
Unique to Azure, Azure Hybrid Benefit (AHB) is a licensing benefit that allows you bringing your existing Windows Server and SQL Server licenses with Software Assurance (SA) to Azure. AHB can bring you up to 85% savings compared to pay-as-you-go pricing in Azure SQL, when combined with reservations savings and extended security updates.
```

Reply:
I did not add any extras about licensing in detail; I just updated options for AHB. Understand rejection. It is better not to say anything than to present only part of the information. How about the link for details (my last sentence) for easy navigation?